### PR TITLE
VEN-336 | Bills card

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: src/common/labelValuePair/LabelValuePair.tsx [hunk 1]
```diff
@@ -5,21 +5,41 @@ import styles from './labelValuePair.module.scss';
...
-      {label}:
-    </span>
-    <span className={styles.value}>{value}</span>
-  </div>
-);
+  align = 'left',
+}: Props) => {
+  let valueArr: React.ReactElement[] = [];
+
+  if (Array.isArray(value)) {
```
Comment (Zeukkari) [inline src/common/labelValuePair/LabelValuePair.tsx]: I don't think this component should be extended to accept array values. The Paragraph component was created for this purpose.

File: src/domain/individualCustomer/billsCard/BillsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+import { useTranslation } from 'react-i18next';
+
+import Card from '../../../common/card/Card';
+import Text from '../../../common/text/Text';
+import Button from '../../../common/button/Button';
+import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
+import Paragraph from '../../../common/paragraph/Paragraph';
+import styles from './billsCard.module.scss';
+
+interface Props {
```
Comment (Zeukkari) [inline src/domain/individualCustomer/billsCard/BillsCard.tsx]: Can we change props naming convention from `interface Props` to `export interface ComponentProps` for compatibility with HDS?

File: src/domain/individualCustomer/billsCard/BillsCard.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  lightingFee,
+  mooringFee,
+  handleShowBill,
+  waterFee,
+  wasteFee,
+  total,
+}) => {
+  const { t, i18n } = useTranslation();
+
+  const formatter = new Intl.NumberFormat(i18n.language || 'fi-FI', {
```
Comment (Zeukkari) [inline src/domain/individualCustomer/billsCard/BillsCard.tsx]: Is default language necessary?

File: src/common/button/Button.tsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ export type Props = {
   variant?: 'contained' | 'outlined' | 'text';
   size?: 'small' | 'standard' | 'large';
   icon?: JSX.Element;
+  className?: string;
```
Comment (Zeukkari) [inline src/common/button/Button.tsx]: Can we stop using this component and use the one from HDS instead?

File: src/domain/individualCustomer/IndividualCustomerPageContainer.tsx [hunk 5]
```diff
@@ -1,23 +1,20 @@
...
-import { useTranslation } from 'react-i18next';
 
 import IndividualCustomerPage from './individualCustomerPage/IndividualCustomerPage';
 import { INDIVIDUAL_CUSTOMER_QUERY } from './queries';
 import { INDIVIDUAL_CUSTOMER } from './__generated__/INDIVIDUAL_CUSTOMER';
 import Card from '../../common/card/Card';
-import Paragraph from '../../common/paragraph/Paragraph';
-import LabelValuePair from '../../common/labelValuePair/LabelValuePair';
-import Text from '../../common/text/Text';
+import BillsCard from './billsCard/BillsCard';
```
Comment (Zeukkari) [inline src/domain/individualCustomer/IndividualCustomerPageContainer.tsx]: CustomerInfoCard and BillsCard look great and IndividualCustomerPageContainer.tsx is improved a lot.

File: src/locales/fi.json [hunk 6]
```diff
@@ -92,6 +92,22 @@
       "phone": "Matkapuhelin",
       "email": "Sähköpostiosoite",
       "remarks": "Huomiot"
+    },
+    "customerBill": {
+      "title": "AVOIMET LASKUT",
```
Comment (Zeukkari) [inline src/locales/fi.json]: This a minor note, but using all capital letters should be moved from translations to styling.

File: src/domain/individualCustomer/customerInfoCard/CustomerInfoCard.tsx [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <Card title={t('individualCustomer.customerInformation.title')}>
+      <Text as="h4" size="m">
+        {t('individualCustomer.customerInformation.applicantPersonalInfo')}
+      </Text>
+      <Paragraph>
+        <LabelValuePair
+          label={t('individualCustomer.customerInformation.firstNames')}
```
Comment (Zeukkari) [inline src/domain/individualCustomer/customerInfoCard/CustomerInfoCard.tsx]: Should this be first name or first names?

Comment (Nizar-Rahme) [reply src/domain/individualCustomer/customerInfoCard/CustomerInfoCard.tsx]: I don't really know 🤔in Abstract it's `Etunimet`

Comment (igordavydsson) [reply src/domain/individualCustomer/customerInfoCard/CustomerInfoCard.tsx]: well, the field is called `firstName` in the backend, but in reality a lot of customers will have all their first names in there

Comment (Zeukkari) [reply src/domain/individualCustomer/customerInfoCard/CustomerInfoCard.tsx]: I'd prefer to follow strictly backend spec over UI design, but I'm willing to let this slide if you prefer to keep it like this.

File: src/domain/individualCustomer/billsCard/BillsCard.tsx [hunk 8]
```diff
@@ -77,22 +78,26 @@ const BillsCard: React.SFC<ComponentProps> = ({
         <LabelValuePair
           align="right"
           label={t('individualCustomer.customerBill.mooring')}
-          value={[mooringFee[0], formatter.format(mooringFee[1])]}
+          value={`${mooringFee[0]}\u00A0\u00A0${formatter.format(
```
Comment (Zeukkari) [inline src/domain/individualCustomer/billsCard/BillsCard.tsx]: This is hard to read. Can we move this formatting logic to a dedicated formatting function?

**Pull request merged**